warplib/ecrecover.cairo

```diff
@@ -10,7 +10,10 @@ from starkware.cairo.common.uint256 import Uint256
 from starkware.cairo.common.cairo_secp.bigint import BigInt3, uint256_to_bigint
 from starkware.cairo.common.cairo_secp.ec import EcPoint
 
-from warplib.maths.eq import warp_eq
+from warplib.maths.eq import warp_eq, warp_eq256
+from warplib.maths.lt import warp_lt256
+
+const SECP256K1_N = 115792089237316195423570985008687907852837564279074904382605163141518161494337
```

Comment:
Hm, this is actually close to 256 bits so it won't fit felt. Too bad there's no compiler warning for truncation.